.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+install:
+  - ps: Start-FileDownload 'https://static.rust-lang.org/dist/rust-nightly-i686-pc-windows-gnu.exe'
+  - rust-nightly-i686-pc-windows-gnu.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
+  - SET PATH=%PATH%;C:\Program Files (x86)\Rust\bin
+  - rustc -V
+  - cargo -V
+  - git submodule update --init --recursive
+
+build: false
+
+test_script:
+  - cargo build
+  - cargo test
```

Comment:
so funny fact https://ci.appveyor.com/project/steveklabnik/rustdoc/build/1.0.2#L142 due to https://github.com/steveklabnik/rustdoc/issues/22

so i can only build

I was going to test rustfmt and npm as well, but given that they should be platform-agnostic, figured it wasn't worth it. maybe they are...